packages/vaadin-combo-box/src/vaadin-combo-box-item.js

```diff
@@ -104,31 +99,16 @@ class ComboBoxItemElement extends ThemableMixin(DirMixin(PolymerElement)) {
   }
 
   static get observers() {
-    return [
-      '_rendererOrItemChanged(renderer, index, item.*)',
-      '_updateLabel(label, _itemTemplateInstance)',
-      '_updateTemplateInstanceVariable("index", index, _itemTemplateInstance)',
-      '_updateTemplateInstanceVariable("item", item, _itemTemplateInstance)',
-      '_updateTemplateInstanceVariable("selected", selected, _itemTemplateInstance)',
-      '_updateTemplateInstanceVariable("focused", focused, _itemTemplateInstance)'
-    ];
+    return ['_rendererOrItemChanged(renderer, index, item.*, selected, focused)', '_updateLabel(label)'];
```

Comment:
Related to the `_updateLabel` comment, let's consider passing `renderer` to it:

```suggestion
    return ['_rendererOrItemChanged(renderer, index, item.*, selected, focused)', '_updateLabel(label, renderer)'];
```